### ISSUE TITLE
Highlighting the active tab in the Libraries' browser

### ISSUE THREAD

**Issue by lochel** [OpenModelica/OpenModelica#7104]
### Description
I usually have several models with the same components or system name open. It woul dmake it easier to see which model I am editing if the active tab is highlighted in the libraries' browser.

![image](https://user-images.githubusercontent.com/8457302/105489584-c0e3fb80-5cb3-11eb-8d7f-6ae8aa289417.png)

### Version and OS
- OMCompiler v1.17.0-dev.354+g9220249be4
- OMSimulator v2.0.0.post281-g1ce5796-linux
- OS: WSL - Ubuntu 20.04

**Comment by adeas31** [comment]
The highlight you see in the `Libraries Browser` is just the item selection highlight regardless of the active tab. I think is also possible to highlight the active tab maybe with some color or border around the item. But I have to warn that this could be annoying to the user as well in a sense that if user has expanded to some item in the libraries hierarchy and then change the active tab then the libraries hierarchy moves the active tab automatically.

**Comment by lochel** [comment]
I agree, I came to the same conclusion after writing the issue. However, highlighting the active tab without changing the view of the libraries browser otherwise would be very nice indeed.

A sort of related idea: it would be good to have a button that reveals the active class in the libraries browser. This is something I usually miss whenever I opened a class using the search bar. Once I found and opened a class it is unnecessary annoying to find it in the library tree.
